src/Razor/src/Microsoft.VisualStudio.LanguageServerClient.Razor/DefaultRazorLanguageServerCustomMessageTarget.cs

```diff
@@ -1140,7 +1140,11 @@ public override Task<ImplementationResult> ImplementationAsync(DelegatedPosition
             request,
             cancellationToken).ConfigureAwait(false);
 
-        if (response is null)
+        // Sometimes web tools will respond with a non-null response, of an array of diagnostics with a single element
```

Comment:
Do we have/can we create an issue against the WebTools folks about this? Returning a null property seems like a weird failure state.

Reply:
I don't know whether web tools is doing something wrong in returning it, the platform is doing something wrong in not handling it, or we're doing something wrong in not filtering it out (and its entirely possible we used to, before moving things around with single server, so none of this might even be new behaviour)